client/src/app/site/pages/meetings/modules/participant-search-selector/components/participant-search-selector/participant-search-selector.component.ts

```diff
@@ -149,9 +151,16 @@ export class ParticipantSearchSelectorComponent extends BaseUiComponent implemen
         }
     }
 
-    public async createNewSelectedUser(username: string): Promise<void> {
-        const newUserObj = await this.userRepo.createFromString(username);
+    public async createNewSelectedUser(name: string): Promise<void> {
+        const newUserObj = await this.userRepo.createFromString(name);
         this.emitSelectedUser({ userId: newUserObj.id, user: newUserObj });
+        const user = await firstValueFrom(
+            this.userRepo.getViewModelObservable(newUserObj.id).pipe(filter(user => !!user))
+        );
+        this.snackBar.open(
+            `A user with the username '${user.username}' and the surname '${user.last_name}' was created.`,
+            `Ok`
```

Comment:
This is not translatable.

Reply:
Use instant and replace to add a translation here.
The keywords USERNAME and LAST_NAME are used in the message.